src/Service/ActivityPubManager.php

```diff
@@ -157,38 +157,40 @@ public function findActorOrCreate(?string $actorUrlOrHandle): null|User|Magazine
             return $this->userRepository->findOneBy(['username' => $name]);
         }
 
+        $user = $this->userRepository->findOneBy(['apPublicUrl' => $actorUrl]);
+        if ($user instanceof User) {
+            $this->logger->debug('found remote user for url "{url}" in db', ['url' => $actorUrl]);
+            if ($user->apId && (!$user->apFetchedAt || $user->apFetchedAt->modify('+1 hour') < (new \DateTime()))) {
+                $this->dispatchUpdateActor($user->apProfileId);
+            }
+
+            return $user;
+        }
+        $magazine = $this->magazineRepository->findOneBy(['apPublicUrl' => $actorUrl]);
```

Comment:
Same thing here, I reverted `'apPublicUrl'` to `'apProfileId'`... what was the reason for changing this?